Лабораторная работа #2

Modiss3 opened the pull request.

Modiss3 committed:
Лабораторная работа #2

mixeme commented on Лабораторная работа #2/task_1.py:
```diff
@@ -0,0 +1,19 @@
+month_counter = 0 #
+
+while not_in_debt:
+    spend += spend * increase * bool(month_counter) # Перевод счетчика месяцев в булево значение, т.к. в нулевой итерации (первом месяце) увеличения трат нет, а во вслех последующих есть
```
Очень странный ход с умножением на bool. Это точно было необходимо?

mixeme replied:
Необходимость данного решения осталась без пояснений.

mixeme commented on Лабораторная работа #2/task_2.py:
```diff
@@ -0,0 +1,14 @@
+    money_capital += spend - salary
+
+
+print(f"Подушка безопасности, чтобы протянуть {months} месяцев без долгов: {money_capital:.0f}")
```
Предполагалось использование функции округления.

mixeme commented:
Поправить решения.

Modiss3 committed:
Обновление решения

Изменения:
1. В задании #2 округление финансовой подушки теперь происходят в большую сторону.

Modiss3 commented:
Решение обновлено.

mixeme reviewed (commented):
Необходимость данного решения осталась без пояснений. Оценку не менял.
